meta-openpli/recipes-multimedia/gstreamer/gstreamer1.0_git.bb

```diff
@@ -7,7 +7,9 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=6762ed442b3822387a51c92d928ead0d \
 SRC_URI = " \
 	git://anongit.freedesktop.org/gstreamer/gstreamer;branch=master;name=base \
 	git://anongit.freedesktop.org/gstreamer/common;destsuffix=git/common;name=common \
+	file://deterministic-unwind.patch \
```

Comment:
ERROR: gstreamer1.0-1.12.00+gitAUTOINC+7854a65978-r0 do_fetch: Fetcher failure for URL: 'file://deterministic-unwind.patch'. Unable to fetch URL from any source.